src/SFA.DAS.RoatpGateway.Web.UnitTests/Services/PeopleInControl/Orchestrator/PeopleInControlHighRiskTests.cs

```diff
@@ -124,28 +130,34 @@ public void check_people_in_control_high_risk_psc_details_from_submitted_are_ret
             Assert.AreEqual(viewModel.PscData.PeopleInControl.First().MonthYearOfBirth, pcsFromSubmitted.First().MonthYearOfBirth);
         }
 
-        [Test]
-        public void check_people_in_control_trustee_details_from_submitted_are_returned()
-        {
-            var trusteesFromSubmitted = new List<PersonInControl>
-            {
-                new PersonInControl
-                {
-                    Name = PersonInControlName+ TrusteesPostfix,
-                    MonthYearOfBirth = PersonInControlDob + TrusteesPostfix
-                }
-            };
-            _organisationSummaryApiClient.Setup(x => x.GetTrusteesFromCharityCommission(_applicationId)).ReturnsAsync(trusteesFromSubmitted);
+         [Test]
+         public void check_people_in_control_trustee_details_from_submitted_are_returned()
+         {
+             var trusteesFromSubmitted = new List<PersonInControl>
+             {
+                 new PersonInControl
+                 {
```

Comment:
What is this post fix for?

Reply:
The setup is spurious, I lifted from another test. I'll clean it up.